packages/taquito-local-forging/src/uint8array-consumer.ts

```diff
@@ -8,11 +9,11 @@ export class Uint8ArrayConsumer {
       );
       return new Uint8ArrayConsumer(arr);
     } else {
-      throw new InvalidHexStringError(lowHex);
+      throw new InvalidHexStringError(`The hex string '${lowHex}' is invalid`);
     }
   }
 
-  constructor(private readonly arr: Uint8Array, private offset: number = 0) {}
+  constructor(private readonly arr: Uint8Array, private offset: number = 0) { }
```

Comment:
I think the previous was correct (no whitespace in between curly brackets)

Reply:
yea i need to change my formatter since it is default for that 

only spaces we want between curly braces is with variables inbetween right? for the braces that are opened and closed on the same line